examples/mie_sphere/mie_sphere_test.m

```diff
@@ -20,7 +24,8 @@
 % callback settings
 opt.callback_interval   = 8;
 opt.callback            = @Simulation.abs_crossimage_callback;
-
+%Use no callback for a fair comparison with the Mex code
```

Comment:
Remember to include it in the README. And have a default option in every test (of no callback or some callback?).